internal/openshift/controller.go

```diff
@@ -31,7 +32,8 @@ type Controller interface {
 	GetUser(ns string) model.User
 }
 
-type OpenShiftController struct {
+// ControllerImpl watches OpenShift cluser for changes and implements Controller.
+type ControllerImpl struct {
```

Comment:
I did the rename to 'controller' locally and did not see any problems. Not sure what problem you had. If you want to keep 'ControllerImpl ', please make it at least 'controllerImpl' since we don't want to export this type.

Reply:
`controller` is a variable in controller_test.go. I will make it `controllerImpl`.

Reply:
>I will make it controllerImpl

+1